src/set_theory/game/domineering.lean

```diff
@@ -95,7 +95,7 @@ lemma move_left_card {b : board} {m : ℤ × ℤ} (h : m ∈ left b) :
   finset.card (move_left b m) + 2 = finset.card b :=
 begin
   dsimp [move_left],
-  rw finset.card_erase_of_mem,
+  rw @finset.card_erase_of_mem _ (finset.erase b m) _ _,
```

Comment:
Do you know why these needed to change?

Reply:
I have no idea in particular but it must be due to the changed imports in `data.finset.lattice`. @YaelDillies has suggested doing a `lex` refactor at some point in which case maybe this band-aid solution could be OK?

Reply:
I ended up just golfing away these proofs in #11298 anyway... It seems the reason this gets stuck is that lean doesn't like applying lemmas about `finset` to `board`.